src/main/kotlin/brs/db/sql/SqlDb.kt

```diff
@@ -133,6 +129,8 @@ internal class SqlDb(private val dp: DependencyProvider) : Db {
                 flywayBuilder.locations("classpath:/db/migration_sqlite")
                 runFlyway = true
                 config.isAutoCommit = true
+                config.addDataSourceProperty("journal_mode", "WAL")
```

Comment:
Huh, I had no idea this works... Unfortunately documentation around data source properties is very poor and I did not see mentioned that you could do this... Thanks!